src/pages/Login.tsx

```diff
@@ -78,10 +78,9 @@ const Login: React.FC = () => {
               /> */}
           </div>
           <div id="chimple-brand-text1">Welcome to Chimple!</div>
-          <p id="chimple-brand-text2">
-            Discovering the joy of learning with Chimple- where curiosity meets
-            education!
-          </p>
+          <p id="chimple-brand-text2">&nbsp;&nbsp;Discovering the joy of learning with</p>
```

Comment:
Try to make the text center using `css` not by using `&nbsp`, because we have to support multiple languages not just English

Reply:
There is no &nbsp as I have removed it after the last PR review.

Reply:
I can still see `&nbsp` in the code. Check the above code([line number 81](https://github.com/chimple/cuba/pull/292/files#diff-a66bb9a4e6565ac9617171d098ce0cb1ce99d28b86ab8670f7e38396b40fbb7bR81))

Reply:
Please check now.